projects/packages/jetpack-mu-wpcom/src/features/wpcom-options-general/options-general.php

```diff
@@ -40,7 +40,7 @@ function wpcom_fiverr() {
 function wpcom_site_management_panel_link() {
 	?>
 	<a href="https://wordpress.com/sites/settings/site/<?php echo esc_attr( wpcom_get_site_slug() ); ?>">
-		<?php esc_html_e( 'Manage your WordPress.com site settings, including site visibility, and more.', 'jetpack-mu-wpcom' ); ?>
+		<?php esc_html_e( 'Manage site visibility, gift subscriptions, ownership, and other site tools on WordPress.com ↗', 'jetpack-mu-wpcom' ); ?>
```

Comment:
```suggestion
		<?php esc_html_e( 'Manage site visibility, ownership, and other site tools on WordPress.com ↗', 'jetpack-mu-wpcom' ); ?>
```

Let's omit the gift subscription thing, it's not a big feature (covered by "other site tools") and may not be around forever.

Reply:
> may not be around forever

Hmm, good point.
For context (just in case!), I included this because of the recent conversation about making the gift subscription opt-in: p1739023467265609-slack-C07H21B2W59.
Like, If you notice an unwanted prominent banner on your site, you might want to find out how to configure that easily.

Reply:
nit: Would it be better to use the [external icon](https://developer.wordpress.org/resource/dashicons/#external)?

Reply:
Ignore me since we use the same one on this modal... 🙈

<img src="https://github.com/user-attachments/assets/c1ee131b-033b-4e3d-979d-25162302626c" width="300" />

Reply:
Could we use the [ExternalLink component](https://wordpress.github.io/gutenberg/?path=/docs/components-externallink--docs) here?

That handles the icon and correctly underlines the content. (The text should have an underline, but the arrow should not.)

Reply:
>Ignore me since we use the same one on this modal... 🙈

Yes, and it's used in profile.php, too: https://github.com/Automattic/jetpack/pull/39636#discussion_r1787079928

>Could we use the [ExternalLink component](https://wordpress.github.io/gutenberg/?path=/docs/components-externallink--docs) here?

That would be ideal, but integrating it into wp-admin is tricky since it's not built with React. We should aim for consistency across all external links. I'll create an issue for this.

Possibly related: https://github.com/Automattic/wp-calypso/issues/98031#issuecomment-2646141592